frontend/src/lib/components/Errors/ErrorDisplay.tsx

```diff
@@ -138,8 +135,15 @@ export function getExceptionPropertiesFrom(eventProperties: Record<string, any>)
         $level,
     } = eventProperties
 
+    let $exception_type = eventProperties.$exception_type
+    let $exception_message = eventProperties.$exception_message
+    let $exception_synthetic = eventProperties.$exception_synthetic
     let $exception_stack_trace_raw = eventProperties.$exception_stack_trace_raw
     let $exception_list = eventProperties.$exception_list
+
+    // TODO: I want to remove stack_trace_raw completely - this will break old existing js events, but the same
+    // will happen again anyway when the rust pipeline is ready. Since no users are using this(right?)- seems alright to go for it.
```

Comment:
As long as we're comfortable that the Sentry integration doesn't rely on it then 👍

Reply:
Yep will do rust separately in a follow up.

And yeah the sentry integration is more for sending events into posthog, using sentry exception capture - I've already updated it to use exception_list in the last js PR, and I have failovers for type and message in this PR, so yeah that should still work fine 👍 - didn't use stack trace raw.